index.rst

```diff
@@ -41,7 +41,8 @@ visit our `gallery </docs/dev/auto_examples>`__.
 
         from skimage import data, io, filters
 
-        image = data.coins() # or any NumPy array!
+        # ... or any other NumPy array!
+        image = data.coins()
```

Comment:
I'd reverse this and put the comment underneath — it reads much more like English.